src/selectors/index.js

```diff
@@ -1,7 +1,57 @@
-export function sectionsSelector(state) {
+import { meetsDuringDow, meetsDuringHour } from 'util/time';
+
+function getMatchingClasses(classes, hour, dow) {
+  return classes.filter(
+    (sectionOrAssociatedClass) => {
+      const eventObj = sectionOrAssociatedClass.get('event').toJS();
+      return meetsDuringDow(eventObj, dow) && meetsDuringHour(eventObj, hour);
+    },
+  );
+}
+
+function findMatchingClass(classes, hour, dow) {
+  return classes.find(
+    (sectionOrAssociatedClass) => {
+      const eventObj = sectionOrAssociatedClass.get('event').toJS();
+      return meetsDuringDow(eventObj, dow) && meetsDuringHour(eventObj, hour);
+    },
+  );
+}
+
+/* schedule */
```

Comment:
No need for this to be a multi line comment. Can we make this `// Schedule` instead? (Same with browse and search)